web/src/components/Editor/Interpolation/Interpolation.tsx

```diff
@@ -0,0 +1,47 @@
+import CodeMirror from '@uiw/react-codemirror';
+// import {autocompletion} from '@codemirror/autocomplete';
```

Comment:
Commented this for the moment, it will be re-added once we integrate the editors with the rest of the app